bundle/edu.gemini.horizons.api/src/test/scala/edu/gemini/horizons/server/backend/HS2Spec.scala

```diff
@@ -177,4 +183,26 @@ object HS2Spec extends Specification with ScalaCheck {
 
   }
 
+  "ephemeris element lookup" should {
+
+    // sanity check with a known ephemeris, to catch changes in output format
+    "return a correct ephemeris for Halley (comet)" in {
+      val result = runLookupE(HD.Comet("1P"), 10).toOption.map(_.toList.map(_.toString.replace('\t', ' ')))
+      result must_== Some(List(
+        "(1675184400000,2023-Jan-31 17:00:00 UTC 08:17:44.423 +02:15:52.58 -4.29131 1.18328 -1.0 25.535)",
+        "(1676748600000,2023-Feb-18 19:30:00 UTC 08:15:43.624 +02:25:31 -3.88565 1.474407 -1.0 25.54)",
+        "(1678312800000,2023-Mar-08 22:00:00 UTC 08:14:01.534 +02:36:42.31 -3.10267 1.605414 2.302 25.55)",
+        "(1679877000000,2023-Mar-27 00:30:00 UTC 08:12:47.94 +02:48:16.16 -2.01609 1.573012 1.192 25.565)",
+        "(1681441200000,2023-Apr-14 03:00:00 UTC 08:12:09.054 +02:59:03.78 -0.7403 1.389621 2.015 25.583)",
+        "(1683005400000,2023-May-02 05:30:00 UTC 08:12:07.022 +03:08:06.46 0.577434 1.085775 -1.0 25.601)",
+        "(1684569600000,2023-May-20 08:00:00 UTC 08:12:40.294 +03:14:39.1 1.767347 0.70198 -1.0 25.619)",
+        "(1686133800000,2023-Jun-07 10:30:00 UTC 08:13:44.373 +03:18:12.92 2.707565 0.271757 -1.0 25.634)",
+        "(1687698000000,2023-Jun-25 13:00:00 UTC 08:15:12.407 +03:18:36.17 3.351105 -0.17043 20.829 25.646)",
+        "(1689262200000,2023-Jul-13 15:30:00 UTC 08:16:56.008 +03:15:51.91 3.688445 -0.58832 1.412 25.653)",
+        "(1690826400000,2023-Jul-31 18:00:00 UTC 08:18:45.968 +03:10:17.27 3.734402 -0.95298 1.306 25.655)"
```

Comment:
I wonder if these change when REL-4210 is merged.

Reply:
They might change, in which case you can print the result out and paste it back into the test. I had to do this a few times as I was messing around.

Reply:
Let's merge that first and I'll update the test.

Reply:
Ok, thanks done.